src/server/models/list.ts

```diff
@@ -43,14 +42,15 @@ export async function findListByCountryAndType(country: CountryName, type: Servi
   }
 }
 
-export async function findListByAnnualReviewDate(annualReviewStartDate: Date): Promise<Result<List[]>> {
+export async function findListByAnnualReviewDate(annualReviewStartDate: Date, today: Date): Promise<Result<List[]>> {
   try {
     logger.debug(`searching for lists matching date [${annualReviewStartDate}]`);
 
     const result = (await prisma.list.findMany({
       where: {
         nextAnnualReviewStartDate: {
```

Comment:
A couple of things here:

1. I think this condition needs to change as it needs to pull out lists where the nextAnnualReviewStartDate is between today and one month in the future so it can send the annual review one month warning email.  
2. the annualReviewStartDate param is actually [calculated](https://github.com/UKForeignOffice/lists/pull/601/files#diff-ac6f76c119a96d48de4d9713598c00f293f63770102a4186926d55a4a55d5cbaR65) as one month before today which won't work if we're trying to pull out lists coming up for annual review in one month.

## Suggestions
method signature: findListByAnnualReviewDate(annualReviewStartDate: Date)

```suggestion
        nextAnnualReviewStartDate: {
            gte: today,
            lt: annualReviewStartDate, // you could substitute this with startOfDay(addMonth(1, today))
        }
```

```suggestion
            history: {
              some: {
                type: "PUBLISHED",
                time: {
                  gte: annualReviewStartDate,
                  lte: today,
                },
              },
            },
	```